sim/core/aura.go

```diff
@@ -286,24 +288,16 @@ func (unit *Unit) GetOrRegisterAura(aura *Aura) *Aura {
 }
 
 func (at *auraTracker) GetAurasWithTag(tag string) []*Aura {
-	if auras, ok := at.aurasByTag[tag]; ok {
-		return auras
-	} else {
-		return []*Aura{}
-	}
+	return at.aurasByTag[tag]
```

Comment:
Returning the nil slice is more shorter and more idiomatic.

Reply:
:+1: